.github/workflows/node.js.yml

```diff
@@ -0,0 +1,42 @@
+name: Node
+
+on:
+  pull_request:
+  push:
+    branches-ignore: ["dependabot/**"]
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    name: ${{ matrix.NPM_SCRIPT }} - ${{ matrix.NAME }} (${{ matrix.NODE_VERSION }})
+    strategy:
+      fail-fast: false
+      matrix:
+        NAME: ["Node"]
+        NODE_VERSION: [20.x]
+        NPM_SCRIPT: ["test"]
```

Comment:
No need to use a matrix here. I suggest using `env` to set the node version, setting the workflow name statically. `NPM_SCRIPT` is already unused below.

Reply:
I've tried to switch to `env` and it looks similar to what you did in jQuery UI but it just doesn't work. I have a test PR at https://github.com/mgol/jquery-ui-themeroller/pull/2 and I am getting the following error:
```
The workflow is not valid. .github/workflows/node.js.yml (Line: 17, Col: 11): Unrecognized named-value: 'env'. Located at position 1 within expression: env.NODE_VERSION
```
That line is literally:
```
    name: test (${{ env.NODE_VERSION }})
```
Any ideas?

Reply:
Oh; apparently, env variables cannot be used outside `steps`. I removed the usage from that single line and it's green now: https://github.com/mgol/jquery-ui-themeroller/actions/runs/8584283401/job/23524385015?pr=2